frontend/src/main/web/src/components/Flex/readme.md

```diff
@@ -0,0 +1,2 @@
+For a more comprehensive guide on how Flexbox works, check out
+[A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).
```

Comment:
Its a component which generates Flexboxe in `div` with predefined styles. We use this component directly.